psamm_import/importer.py

```diff
@@ -175,14 +175,14 @@ def reactions_to_files(model, dest, yaml_args, exchange, split_subsystem):
     """Turns the reaction subsystems into their own files.
 
     If a subsystem has a number of reactions over the threshold, it gets its
-    own yaml file. All other reactions, those that don't have a subsystem or
+    own YAML file. All other reactions, those that don't have a subsystem or
     are in a subsystem that falls below the threshold, get added to a common
     reaction file.
 
     Args:
-        model: :class:psamm_import.model.MetabolicModel.
+        model: `:class:psamm_import.model.MetabolicModel`.
```

Comment:
Go to the Sphinx documentation and make sure you understand the syntax. This is still incorrect.